pacifica/cartd/utils.py

```diff
@@ -106,6 +107,8 @@ def check_space_requirements(self, cart_file, mycart, size_needed, deleted_flag)
         space. It will delete a cart first, then call  itself
         until either there is enough space or there is no carts to delete
         """
+        if not self._lru_purge:
```

Comment:
@GitBytes this check short circuits the function you want.

Reply:
@dmlb2000 Ah ok, I didn't read it quite right then. Cool! Thanks!